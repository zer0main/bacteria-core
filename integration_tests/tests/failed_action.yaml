board_width: 11
board_height: 11
moves_number: 1
units:
    - coordinates: [10, 10]
      mass: 15
      direction: 1
      team: 0
      instruction: 0
