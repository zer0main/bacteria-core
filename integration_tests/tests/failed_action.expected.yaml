board_width: 11
board_height: 11
units:
    - coordinates: [10, 10]
      mass: 5
      direction: 1
      team: 0
      instruction: 0
